content/php-client/resources/reference-entity/media-files.md

```diff
@@ -0,0 +1,23 @@
+### Reference entity media file
+
+::: warning
+This resource is only available in the [Entreprise Edition](https://www.akeneo.com/enterprise-edition/).
+:::
+
+#### Download the media file associated to a reference entity or a record
+
+```php
+$client = new \Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$mediaFileResponse = $client->getReferenceEntityMediaFileAPi()->download('images/kartell.png');
+
+file_put_contents('/tmp/kartell.png', $mediaFileResponse->getBody()->getContents());
+```
+
+#### Create a new media file for a reference entity or a record
+
+```php
+$client = new \Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$mediaFileCode = $client->getReferenceEntityMediaFileAPi()->create('/tmp/kartell.png');
```

Comment:
I don't like the naming of this API, we will see how to refactor the whole client later on.